xray/src/generation.rs

```diff
@@ -315,11 +315,16 @@ pub fn build_parent(children: &[Option<image::RgbImage>]) -> image::RgbImage {
         }
     }
     let child_size_px = child_size_px.expect("No children passed to 'build_parent'.");
-    let mut large_image = image::RgbImage::from_pixel(
+    let mut large_image = image::RgbaImage::from_pixel(
         child_size_px * 2,
         child_size_px * 2,
-        image::Rgb {
-            data: [255, 255, 255],
+        image::Rgba {
+            data: [
+                TRANSPARENT.to_u8().red,
```

Comment:
Can you make it a comment line option whether to make the background white or transparent?